OpenSim/Common/Reporter.h

```diff
@@ -177,7 +191,16 @@ OpenSim_DECLARE_CONCRETE_OBJECT_T(TableReporter_, InputT, Reporter<InputT>);
               const auto& value = chan.getValue(state);
               result[idx] = value;
         }
-        const_cast<Self*>(this)->_outputTable.appendRow(state.getTime(), result);
+        try {
+            const_cast<Self*>(this)->_outputTable.appendRow(state.getTime(),
+                                                            result);
+        } catch(const InvalidTimestamp&) {
+            OPENSIM_THROW(Exception,
+                          "Attempting to update reporter with rows having "
+                          "invalid timestamps. Hint: If running simulation in "
```

Comment:
What is the default "InvalidTimestamp" exception message? Does it tell the user what makes a timestamp invalid? If the only requirement is for the time to be non-decreasing, perhaps just say that here and/or report the current and previous times; otherwise, maybe the exception should be re-thrown and also caught elsewhere to get the default message as well—though I don't know whether that is good software design.